modules/store/src/runtime_checks.ts

```diff
@@ -0,0 +1,104 @@
+import { isDevMode, Provider } from '@angular/core';
+import {
+  stateSerializationCheckMetaReducer,
+  actionSerializationCheckMetaReducer,
+  immutabilityCheckMetaReducer,
+} from './meta-reducers';
+import { RuntimeChecks, MetaReducer } from './models';
+import {
+  _USER_RUNTIME_CHECKS,
+  _ACTIVE_RUNTIME_CHECKS,
+  META_REDUCERS,
+} from './tokens';
+
+export function createActiveRuntimeChecks(
+  runtimeChecks?: Partial<RuntimeChecks>
+): RuntimeChecks {
+  if (isDevMode()) {
+    if (runtimeChecks === undefined) {
+      console.warn(
+        '@ngrx/store: we added immutability and serializability runtime checks in @ngrx/store.\n' +
```

Comment:
I think we should keep this message short and add information about the checks to a page we link to. Another idea is to make this a comment for the `RootStoreConfig` so it shows up in their IDE and makes the console less noisy.

Reply:
Yea, you're right 😅 the link to a page is a good idea 👍 
My intention was to log it so it will certainly be visible, but this can quickly become annoying. It will also log out the messages in people's tests... On the other hand I'm just afraid this change will be unnoticed by teams if we put in a comment on `RootStoreConfig`. This log can be disabled by providing a config, this can just be an empty object.

Reply:
👍